### DISCUSSION TITLE
how to support intel ax210

### DISCUSSION THREAD

**Discussion by guokaikaio** [osy86/HaC-Mini#1002] [Q&A] [answered]
the new version for itlwm support ax210, but we not now

**Comment by ivan-ef** [comment]
> the new version for itlwm support ax210, but we not now

I heard nuc CPU missing some blocks that support ax210. Are you sure it should works on NUC?

**Reply by guokaikaio** [reply]
ax201 need cpu support,but ax 210 not

**Comment by Medizinmann-MD** [comment, marked as answer]
> the new version for itlwm support ax210, but we not now

What version of HacMini.pkg do you use and what version of MacOS?

AFAIR OpenIntelWireless isn't included at all in the latest build as it doesn't work well with Ventura.

You could install the latest OpenIntelWireless by yourself - but be carefull - I would advise to make an image/carbon copy of your drive and experiment with the copy!

And user ivan-ef has a valid point - does ax210 even work with the NUC Hades Canyon - does ist work with Windows?

**Reply by guokaikaio** [reply]
i can install openitelwireless by myself ,as you can see, it doest work welll with ventura,so i understand why dont include new version openintelwireless . and thank you!
